### PR TITLE
[NUI] Add Tizen.NUI.CommonUI project

### PR DESCRIPTION
**Pull request by Lynettejin90**
### API Changes ###

 - ACR:

### CODE REVIEW

File: src/Tizen.NUI.CommonUI/FriendAssembly.cs [hunk 1]
```diff
@@ -0,0 +1,25 @@
+using System.Runtime.CompilerServices;
+
+[assembly: InternalsVisibleTo("Tizen.FH.NUI, PublicKey=0024000004800000940000000602000000240000525341310004000001000100edd17ed375b5865e35ee337fe9e90ef0fb0c258c091874889540bb4eaeedc992755ec0e0a6deadfbfbb685bddfc99a850418e94b6a9e831fb47bcc52be1a5110dbc141943b51ebefe6daf4e415dc430c6b3f10175188db4b3dc0073d662318fe8463dfdb22aaa3e8e9aa97313fb4b3b157cf32c9b6b340dab9812fe97fd2d6c4")]
+
+namespace Tizen.NUI.CommonUI
+{
+    public class TestPublicClass
```
Comment (m-macias) [inline src/Tizen.NUI.CommonUI/FriendAssembly.cs]: What is the purpose of this class and why it is public?

File: src/Tizen.NUI.CommonUI/Controls/Button.cs [hunk 2]
```diff
@@ -0,0 +1,1503 @@
...
+        }
+        /// <summary>
+        /// Get Button attribues.
+        /// </summary>
+        /// <since_tizen> 6 </since_tizen>
+        /// This will be public opened in tizen_5.5 after ACR done. Before ACR, need to be hidden as inhouse API.
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        protected override Attributes GetAttributes()
+        {
+            return new ButtonAttributes();
```
Comment (huiyueun) [inline src/Tizen.NUI.CommonUI/Controls/Button.cs]: Is it correct to create a new Attribute?

Comment (rabbitfor) [reply src/Tizen.NUI.CommonUI/Controls/Button.cs]: I think **GetAttributes**() is called when Control initialize its **attributes**. 
So returning new XXXAttriubutes seems to be correct.

File: src/Tizen.NUI.CommonUI/Attributes/ImageAttributes.cs [hunk 3]
```diff
@@ -0,0 +1,89 @@
...
+            {
+                Border = attributes.Border.Clone() as RectangleSelector;
+            }
+        }
+        /// <summary>
+        /// Image URL.
+        /// </summary>
+        /// This will be public opened in tizen_5.5 after ACR done. Before ACR, need to be hidden as inhouse API.
+        [EditorBrowsable(EditorBrowsableState.Never)]
+        public StringSelector ResourceURL
```
Comment (huiyueun) [inline src/Tizen.NUI.CommonUI/Attributes/ImageAttributes.cs]: Property name of ImageView is "ResourceUrl".but this is "ResourceURL" 
I think you need to match the property name.

File: src/Tizen.NUI.CommonUI/Controls/Button.cs (hunk 2 above)
Comment (Lynettejin90) [reply src/Tizen.NUI.CommonUI/Controls/Button.cs]: yes, it is correct.

**Pull request merged**